docs/tools/vdb_table/data/pgvecto_rs.json

```diff
@@ -0,0 +1,181 @@
+{
+    "name": "pgvecto.rs",
+    "links": {
+      "docs": "https://docs.pgvecto.rs/",
+      "github": "https://github.com/tensorchord/pgvecto.rs",
+      "website": "https://docs.pgvecto.rs/",
+      "vendor_discussion": "",
+      "poc_github": "https://github.com/kemingy",
+      "slug": "pgvecto.rs"
+    },
+    "oss": {
+      "support": "full",
+      "source_url": "",
+      "comment": ""
+    },
+    "license": {
+      "value": "Apache-2.0",
+      "source_url": "https://github.com/tensorchord/pgvecto.rs/blob/main/LICENSE",
+      "comment": ""
+    },
+    "dev_languages": {
+      "value": [
+        "rust"
+      ],
+      "source_url": "",
+      "comment": ""
+    },
+    "vector_launch_year": 2023,
+    "metadata_filter": {
+      "support": "full",
+      "source_url": "",
+      "comment": ""
+    },
+    "hybrid_search": {
+      "support": "full",
+      "source_url": "https://docs.pgvecto.rs/use-case/hybrid-search.html",
+      "comment": ""
+    },
+    "facets": {
+      "support": "",
+      "source_url": "",
+      "comment": ""
+    },
+    "geo_search": {
+      "support": "full",
+      "source_url": "",
+      "comment": "Postgis"
+    },
+    "multi_vec": {
+      "support": "full",
+      "source_url": "",
+      "comment": ""
+    },
+    "sparse_vectors": {
+      "support": "full",
+      "source_url": "https://docs.pgvecto.rs/use-case/sparse-vector.html",
+      "comment": "v0.3.0"
+    },
+    "bm25": {
+      "support": "partial",
+      "source_url": "https://docs.pgvecto.rs/use-case/hybrid-search.html",
+      "comment": "Support scoring by TF, not exactly BM25"
```

Comment:
since it is not BM25, can you change this to `none`?

Reply:
BM25 is a special format of TF-IDF. So I think the TF score should be a "partial" of BM25. What do you think?

Reply:
not really, that would be the case if it was the other way around - if you supported BM25 and the attribute was about TF-IDF. I'll make sure to update pgvector's file to reflect the same as well.

Reply:
I have added another pg extension for the fully BM25 support. PTAL